### ISSUE TITLE
GetServiceAccountIssuerOpenID Unable to deserialize the response

### ISSUE THREAD

**Issue by vturecek** [kubernetes-client/csharp#969]
The methods that deal with Service Account issuer Open ID are unable to deserialize the response in the .NET client:

```
GetServiceAccountIssuerOpenIDConfigurationAsync
GetServiceAccountIssuerOpenIDKeysetAsync
```

The problem seems to be that both of these methods return a `string`. The JSON serializer is unable to deserialize the JSON response to a string type and so you get a JSON exception:

```
Microsoft.Rest.SerializationException: Unable to deserialize the response.\r\n ---\u003E Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: {. Path \u0027\u0027, line 1, position 1.\r\n  
```

I'm currently testing with cluster version 1.22.4, client package version 6.0.28, but even in the latest 8.0.12 package the return type is still a `string`. Is there a way to work around this, like getting the raw response without it trying to deserialize first?

**Comment by vturecek** [comment]
I was able to work around this by using my own class that derives from `k8s.Kubernetes` and overrides this method:

`Task<HttpOperationResponse<T>> CreateResultAsync<T>(HttpRequestMessage httpRequest, HttpResponseMessage httpResponse, bool? watch, CancellationToken cancellationToken)`

It checks if the type T is a string, and if so, doesn't try to deserialize, it just sets the raw string from the response.

I'm not entirely sure why `GetServiceAccountIssuerOpenIDKeysetAsync` returns a string, I'm guessing because there isn't enough info in the swagger spec that this is autogenerated from? We could probably do a couple things though:
1. Update `CreateResultAsync` to check for `string` types as I described above. That will at least make this API work without throwing and will cover any future cases where an API returns a `string`.
2. If there's a way to manually specify the return type for the GetServiceAccountIssuerOpenIDKeysetAsync autogenerated method, the model would be simple since this is returning a list of keys:

```
    public class ServiceAccountIssuerKeySet
    {
        public IEnumerable<Microsoft.IdentityModel.Tokens.JsonWebKey> Keys { get; set; }
    }
```

This works nicely with `System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler` which can take the list in `Keys` directly when validating a token using a set of issuer signing keys (using an autogenerated model here that doesn't derive from Microsoft.IdentityModel.Tokens.SecurityKey or JsonWebKey would be cumbersome for consumers).
